indico/modules/events/editing/templates/management/editable_type.html

```diff
@@ -0,0 +1,9 @@
+{% extends 'events/editing/management/base.html' %}
+
+{% block subtitle %}
+    {% trans t=type|capitalize %} {{ t }} {% endtrans %}
+{% endblock %}
+
+{% block content %}
+    <div id="editable-type" data-event-id="{{ event.id }}" data-editable-type="{{ type }}"></div>
+{% endblock %}
```

Comment:
Missing trailing linebreak. Set `files.insertFinalNewline` to true in your vscode config to add it automatically in all files you save.

(same problem in filetypes.html)